src/MagnumPlugins/TinyGltfImporter/Test/TinyGltfImporterTest.cpp

```diff
@@ -286,13 +287,13 @@ constexpr struct {
 } MaterialInvalidData[]{
     {"unknown alpha mode", "unknown alpha mode WAT"},
     {"invalid texture index normalTexture", "normalTexture index 2 out of bounds for 2 textures"},
-    {"invalid texCoord index normalTexture", "multiple texture coordinate sets are not supported"},
+    {"invalid texCoord index normalTexture", "multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them"},
     {"invalid texture index pbrMetallicRoughness", "baseColorTexture index 2 out of bounds for 2 textures"},
     {"invalid texture index pbrSpecularGlossiness diffuse", "diffuseTexture index 2 out of bounds for 2 textures"},
     {"invalid texture index pbrSpecularGlossiness specular", "specularGlossinessTexture index 2 out of bounds for 2 textures"},
-    {"invalid texCoord index pbrMetallicRoughness", "multiple texture coordinate sets are not supported"},
-    {"invalid texCoord index pbrSpecularGlossiness diffuse", "multiple texture coordinate sets are not supported"},
-    {"invalid texCoord index pbrSpecularGlossiness specular", "multiple texture coordinate sets are not supported"},
+    {"invalid texCoord index pbrMetallicRoughness", "multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them"},
+    {"invalid texCoord index pbrSpecularGlossiness diffuse", "multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them"},
+    {"invalid texCoord index pbrSpecularGlossiness specular", "multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them"},
```

Comment:
Since those are now handled in a dedicated test case, remove all four from there (and also from the `material-invalid.gltf` file)